fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/AggregateValueTest.java

```diff
@@ -178,6 +194,23 @@ private Object[] pairsForAvg(Object[] objects) {
                 .toArray();
     }
 
+    private Object[] bitsetForBitMap(Object[] objects) {
+        return Arrays.stream(objects)
+                .map(object -> {
+                    if (object == null) {
+                        return null;
+                    }
+                    BitSet result = new BitSet();
+                    if (object instanceof Integer) {
+                        result.set((int)object);
+                    } else {
+                        result.set(((Long)object).intValue());
+                    }
+                    return result;
+                }) // left for the sum, right for the count
```

Comment:
Is this comment correct? It looks like this is translating an array of numbers to an array of bit sets where these numbers are set?

Reply:
yeah this was a copy paste error. I removed the comment.